mcweb/backend/sources/tasks.py

```diff
@@ -315,3 +316,95 @@ def schedule_scrape_source(source_id, user):
                           remove_existing_tasks=True)
     return return_task(task)
 
+
+DAYS_BACK = 365  # Number of days to look back for story analysis
+MIN_STORY_COUNT = 100  # Minimum number of stories required for a valid source
+START_DATE = dt.datetime(2000, 1, 1) # Possible earliest source publication date
+END_DATE = dt.datetime.now()
```

Comment:
Been a while since I've thought of module var initialisation but shouldn't `END_DATE` be in a method since you want `now()` to be the time the method was called rather than when the module was imported/initialised?